### PR TITLE
Support for Erlang/OTP 1[7-9] only

### PR DESCRIPTION
**Pull request by michalwski**
This PR adds support for Erlang/OTP 19 and drops support for all older versions than 17.

Requires esl/wsecli#10

### CODE REVIEW

**Comment by michalwski** [comment]
This PR is used by esl/MongooseIM#993

File: src/escalus_history_h.erl [hunk 1]
```diff
@@ -10,11 +10,7 @@
 	 handle_event/2,
```
Comment (elvisci) [inline src/escalus_history_h.erl]: Line 10 has a tab at column 0.

**Comment by erszcz** [comment]
Looks good to me!

**Comment by michalwski** [comment]
Travis is green, let's merge this PR.

**Pull request merged**